src/themes/default/views/feed.variables

```diff
@@ -190,6 +190,9 @@
 @invertedLikeHoverColor: @invertedSelectedTextColor;
 @invertedLikeActiveColor: @invertedLikeColor;
 
+@invertedMetadataActionColor: @invertedLightTextColor;
```

Comment:
This color is too dark as it is declared in site.variables as `@invertedLightTextColor: rgba(255, 255, 255, 0.7);`